x/paychan/handler_test.go

```diff
@@ -50,7 +53,7 @@ func TestPaymentChannelHandlers(t *testing.T) {
 						Recipient:    recipient.Address(),
 						SenderPubkey: srcSig.PublicKey(),
 						Total:        dogeCoin(10, 0),
-						Timeout:      1000,
+						Timeout:      weave.AsUnixTime(now.Add(time.Hour)),
```

Comment:
Why don't you define `var inOneHour = weave.AsUnixTime(now.Add(time.Hour))` above (next to `now` definition). Seems you use that everywhere